setup_bd.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+require_once 'config.php';
+
+$sqlQueries = [
+    "CREATE DATABASE IF NOT EXISTS " . DB_NAME
+    ,
+    "USE " . DB_NAME
+    ,
+    "CREATE TABLE IF NOT EXISTS `" . TABLE_NAME_FOR_USERS . "` (
```

Comment:
it's better not to use constants for table name or column, because in future you might need to rename it and you will have troubles